models/user.go

```diff
@@ -10,36 +10,37 @@ import (
 	"golang.org/x/crypto/bcrypt"
 )
 
-// User contains a user account
+// User is our user model for anyone who signs up with Temporal
 type User struct {
 	gorm.Model
-	EthAddress        string         `gorm:"type:varchar(255);unique"`
-	UserName          string         `gorm:"type:varchar(255);unique"`
-	EmailAddress      string         `gorm:"type:varchar(255);unique"`
-	EnterpriseEnabled bool           `gorm:"type:boolean"`
-	AccountEnabled    bool           `gorm:"type:boolean"`
-	APIAccess         bool           `gorm:"type:boolean"`
-	EmailEnabled      bool           `gorm:"type:boolean"`
-	HashedPassword    string         `gorm:"type:varchar(255)"`
-	Credits           float64        `gorm:"type:float"`
-	IPFSKeyNames      pq.StringArray `gorm:"type:text[];column:ipfs_key_names"`
-	IPFSKeyIDs        pq.StringArray `gorm:"type:text[];column:ipfs_key_ids"`
-	IPFSNetworkNames  pq.StringArray `gorm:"type:text[];column:ipfs_network_names"`
+	UserName          string  `gorm:"type:varchar(255);unique"`
+	EmailAddress      string  `gorm:"type:varchar(255);unique"`
+	EnterpriseEnabled bool    `gorm:"type:boolean"`
+	AccountEnabled    bool    `gorm:"type:boolean"`
+	APIAccess         bool    `gorm:"type:boolean"`
+	EmailEnabled      bool    `gorm:"type:boolean"`
+	AdminAccess       bool    `gorm:"type:boolean"`
```

Comment:
perhaps this could be an integer instead for roles?

```
USER_ROLE=0
ADMIN_ROLE=1
...
```

Reply:
Yes, I will create a  ticket to revamp the permission model